activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Add support for [PostgreSQL enums](https://www.postgresql.org/docs/10/static/datatype-enum.html): .
+
+    Creating a new enum type:
+
+        create_enum :ebook_format, [:pdf, :epub]
+
+    Droping an enum:
+
+        drop_enum :ebook_format
+
+    Adding a new value to an existing enum:
+
+        add_value_to_enum :ebook_format, :mobi
```

Comment:
Perhaps naming the method `add_value_to_enum` could be `add_enum_value`?

Reply:
Much better indeed, thanks!